x-pack/plugins/ml/server/models/results_service/results_service.ts

```diff
@@ -605,6 +612,104 @@ export function resultsServiceProvider(mlClient: MlClient) {
     return finalResults;
   }
 
+  async function getDatafeedResultsChartData({
+    jobId,
+    start,
+    end,
+  }: DatafeedResultsChartDataParams): Promise<GetDatafeedResultsChartDataResult> {
+    const finalResults: GetDatafeedResultsChartDataResult = {
+      bucketResults: [],
+      datafeedResults: [],
+    };
+
+    const { getDatafeedByJobId } = datafeedsProvider(client!, mlClient);
+    const datafeedConfig = await getDatafeedByJobId(jobId);
+
+    const {
+      body: { jobs },
+    } = await mlClient.getJobs({ job_id: jobId });
```

Comment:
real nit, but `getDatafeedByJobId` and `getJobs` could be called in parallel.

Reply:
just in case this comment was lost in a different change, I still think putting them in a `Promise.all` is worth it. But it's still only a nitpick, so please ignore if you want.